### PR TITLE
[WIP]Watch mch-image-manifest-xxx configmap

### PR DESCRIPTION
**Pull request by clyang82**
ref: https://github.com/open-cluster-management/backlog/issues/13426

Signed-off-by: clyang82 <chuyang@redhat.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -133,13 +133,25 @@ func main() {
...
+	config.SetImageManifestConfigMapName()
+
 	gvkLabelMap := map[schema.GroupVersionKind]filteredcache.Selector{
 		v1.SchemeGroupVersion.WithKind("Secret"): {
 			FieldSelector: fmt.Sprintf("metadata.namespace==%s", config.GetDefaultNamespace()),
 		},
 		v1.SchemeGroupVersion.WithKind("ConfigMap"): {
 			FieldSelector: fmt.Sprintf("metadata.namespace==%s", config.GetDefaultNamespace()),
 		},
+		v1.SchemeGroupVersion.WithKind("ConfigMap"): {
```
Comment (marcolan018) [inline main.go:151]: can we have two entities for the same resource kind in the Selector?

Comment (clyang82) [reply main.go:151]: That works in my testing. @morvencao thoughts?

Comment (morvencao) [reply main.go:151]: It should not work.
We will lose the first `configmap` setting.
Did you verify the configmap in `open-cluster-management-observability` is still be watching?

**Comment by clyang82** [comment]
close in favor of #611

**Pull request closed without merging**